odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1121,7 +1121,7 @@ def _postprocess_access_rights(self, node, model):
         is_base_model = self.env.context.get('base_model_name', model._name) == model._name
 
         if node.tag in ('kanban', 'tree', 'form', 'activity', 'calendar'):
-            for action, operation in (('create', 'create'), ('delete', 'unlink'), ('edit', 'write')):
+            for action, operation in (('create', 'create'), ('duplicate', 'create'), ('delete', 'unlink'), ('edit', 'write')):
```

Comment:
```suggestion
            for action, operation in (('create', 'create'), ('duplicate', 'copy'), ('delete', 'unlink'), ('edit', 'write')):
```

Reply:
The 'operation' comes from the different access modes of read, write, create, unlink not the BaseModel methods. It'll fail on this assertion if it's not create.

https://github.com/odoo/odoo/blob/f36046fc2b3d032976402e1bb5ba1aaa6db3a490/odoo/addons/base/models/ir_model.py#L1745